### PR TITLE
Update documents in SystemDesign

### PR DESCRIPTION
**Pull request by RYH61**
### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: docs/zh/SystemDesign/DataQuery/AggregationQuery.md [hunk 1]
```diff
@@ -27,68 +27,69 @@
...
+            aggregateStatistics(
+            aggregateResultList, isCalculatedArray, remainingToCalculate, fileStatistics);
+        if (remainingToCalculate == 0) {
+            return;
+        }
+        seriesReader.skipCurrentFile();
+        continue;
+    }
 
-需要注意的是，在对于每一个 result 进行更新之前，需要首先判断其是否已经被计算完（利用 `isCalculatedList` 列表）；每一次更新后，调用 `isCalculatedAggregationResult()` 方法同时更新列表中的布尔值。如果列表中所有值均为 true，即 `remainingToCalculate` 值为 0，证明所有聚合函数结果均已计算完，可以返回。
```
Comment (mychaow) [inline docs/zh/SystemDesign/DataQuery/AggregationQuery.md:75]: Why delete the description here

File: docs/zh/SystemDesign/DataQuery/AggregationQuery.md [hunk 2]
```diff
@@ -27,68 +27,69 @@
...
+            seriesReader, aggregateResultList, isCalculatedArray, remainingToCalculate);
+        if (remainingToCalculate == 0) {
+            return;
+        }
     }
-  }
 }
 ```
 
-在使用 `overlapedPageData` 进行更新时，由于获得每一个聚合函数结果都会遍历这个 batchData，因此需要调用 `resetBatchData()` 方法将指针指向其开始位置，使得下一个函数可以遍历。
```
Comment (mychaow) [inline docs/zh/SystemDesign/DataQuery/AggregationQuery.md:90]: same above

File: docs/zh/SystemDesign/DataQuery/QueryFundamentals.md [hunk 3]
```diff
@@ -47,9 +47,9 @@ TsFile 各级结构在前面的 [TsFile](../TsFile/TsFile.md) 文档中已有介
 ## 顺序和乱序文件的数据特点
 
 对于顺序和乱序文件的数据，其数据在文件中的分部特征有所不同。
-顺序文件的 TimeseriesMetadata 中所包含的 ChunkMetadata 也是有序的，也就是说如果按照 chunkMetadata1, chunkMetadata2 的顺序存储，那么将会保证 chunkMetadata1.endtime <= chunkMetadata2.startTime。
+顺序文件的  TimeseriesIndex 中所包含的  ChunkIndex 也是有序的，也就是说如果按照 ChunkIndex1, ChunkIndex2 的顺序存储，那么将会保证 ChunkIndex1.endtime <= ChunkIndex2.startTime。
```
Comment (mychaow) [inline docs/zh/SystemDesign/DataQuery/QueryFundamentals.md:50]: I do not find the timeseriesindex class, could you confirm it again.

File: docs/zh/SystemDesign/DataQuery/SeriesReader.md [hunk 4]
```diff
@@ -163,55 +163,55 @@ Object[] values = readerByTimestamp.getValueInTimestamp(timestamps, length);
...
 /*
- * 文件层
- */
-private final List<TsFileResource> seqFileResource;
-	顺序文件列表，因为顺序文件本身就保证有序，且时间戳互不重叠，只需使用 List 进行存储
-	
-private final PriorityQueue<TsFileResource> unseqFileResource;
-	乱序文件列表，因为乱序文件互相之间不保证顺序性，且可能有重叠，为了保证顺序，使用优先队列进行存储
+* 文件索引
+*/
```
Comment (mychaow) [inline docs/zh/SystemDesign/DataQuery/SeriesReader.md:167]: this change to be QueryDataResource, not the fileIndex.  FileIndex is the index.

**Review by mychaow — CHANGES_REQUESTED** [review]

File: docs/zh/SystemDesign/DataQuery/QueryFundamentals.md [hunk 5]
```diff
@@ -46,8 +46,7 @@ TsFile 各级结构在前面的 [TsFile](../TsFile/TsFile.md) 文档中已有介
 
 ## 顺序和乱序文件的数据特点
 
-对于顺序和乱序文件的数据，其数据在文件中的分部特征有所不同。
-顺序文件的 TimeseriesMetadata 中所包含的 ChunkMetadata 也是有序的，也就是说如果按照 chunkMetadata1, chunkMetadata2 的顺序存储，那么将会保证 chunkMetadata1.endtime <= chunkMetadata2.startTime。
+对于顺序和乱序文件的数据，其数据在文件中的分部特征有所不同。 顺序文件的 TimeseriesMetadata 中所包含的 ChunkMetadata 也是有序的，也就是说如果按照  chunkMetadata1, chunkMetadata2 的顺序存储，那么将会保证 chunkMetadata1.endtime <= chunkMetadata2.startTime。
```
Comment (mychaow) [inline docs/zh/SystemDesign/DataQuery/QueryFundamentals.md:49]: Why change two line into one line?

**Review by mychaow — CHANGES_REQUESTED** [review]

**Review by tsunghanjacktsai — APPROVED** [review]

**Review by mychaow — APPROVED** [review]
LGTM

**Pull request merged**